api/server/request_controller.go

```diff
@@ -0,0 +1,117 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/gin-gonic/gin"
+	"github.com/iron-io/functions/api"
+	"github.com/iron-io/runner/common"
+)
+
+type RequestController interface {
+	AppName() string
```

Comment:
@treeder @Gouthamve 
Another use cases - if have a header that represents a user ID (X-Project-ID), and i want to restrict user from accessing other apps by passing list of apps (that are stored in DB, in mapping user_id ---> app) to include/exclude. So that'll be very useful with App(s) objects (at least i suppose so).